core/modules/dashboard/includes/block.taxonomy.inc

```diff
@@ -0,0 +1,154 @@
+<?php
+/**
+ * @file
+ *
+ * Dashboard "Taxonomy" block. Shows statistics and links to manage terms in
+ * vocabularies on the site.
+ */
+class DashboardTaxonomyBlock extends Block {
+
+  /**
+   *  Sets block subject on block view.
+   */
+  function getTitle() {
+    return !empty($this->settings['title']) ? check_plain($this->settings['title']) : t('Categories');
+  }
+
+  /**
+   * 'Admin title' callback for the content type.
+   */
+  function getAdminTitle() {
+    return t('Dashboard manage taxonomy');
```

Comment:
...we aren't using "Dashboard" in the `getAdminTitle()` for the rest of the blocks.

Reply:
yep. fixed.